Detectors/ITSMFT/ITS/tracking/include/ITStracking/Configuration.h

```diff
@@ -106,6 +106,8 @@ struct VertexingParameters {
   int maxTrackletsPerCluster = 2e3;
   int phiSpan = -1;
   int zSpan = -1;
+
+  int nThreads = 1;
```

Comment:
could you point me on where this template is instantiated?

Reply:
Currently we have in the `*TrackerSpec.cxx` a call to `getGlobalConfiguration()` defined here: https://github.com/AliceO2Group/AliceO2/blob/dfae99cd3b675aa6abd45a6383c776862b8e0bb5/Detectors/ITSMFT/ITS/tracking/src/Vertexer.cxx#L56
That resets config values to `VertexerParamConfig` ones (this are the one that can be overridden by `--configKeyValues`).
Then during vertexerTraits initialisation the vertexer is configured again on the same values, this should be probably removed.
I'll have a look in next iteration.

Vertexer/Tracker configuration is a bit cumbersome, probably because of the GPU compatibility (ConfigParams depends on ROOT things).
I will improve that.

Reply:
sorry, I put the link on the wrong line, I meant this template: https://github.com/AliceO2Group/AliceO2/blob/ed25a1e47c5fd105013061949b6fc2582396ab1c/Detectors/ITSMFT/ITS/tracking/include/ITStracking/Configuration.h#L34-L48

Reply:
Ah, that is not something I am aware of, according to git blame it's 4 years old. Probably dead code (?).
Will check and dispose of that, in case.